src/main/java/no/nav/foreldrepenger/fpformidling/domene/beregningsgrunnlag/AktivitetStatus.java

```diff
@@ -27,6 +27,9 @@ public enum AktivitetStatus implements Kodeverdi {
     TTLSTØTENDE_YTELSE("TY"),
     VENTELØNN_VARTPENGER("VENTELØNN_VARTPENGER"),
 
+    //Intern status brukt i formidling for å representere en regelstatus som ikke er definert i kodeverket
+    ARBEID_UNDER_AAP("ARBEID_UNDER_AAP"),
```

Comment:
Trenger man denne? Kan man ikke bare sjekke på OpptjeningAktivitetType? Antar etterlønn / sluttpakke er løst slik

Reply:
Det er sikkert andre måter å løse det på, men for å unngå en del tekst i brevet som gjelder vanlig arbeidstakerstatus eller frilanserstauts så måtte jeg definere en egen slik at jeg skal skille denne fra de. Jeg kan definere en egen boolean, men da blir det mer logikk i malen for å unngå det som ikke skal gjelde, så derfor valgte jeg denne løsningen